### PR TITLE
AAP-30807 Commented out, and updated text for the 2.5 release. (#1929)

### PR DESCRIPTION
**Pull request by rogrange**
* AAP-30807 Commented out, and updated text for the 2.5 release.

### CODE REVIEW

File: downstream/assemblies/platform/assembly-disconnected-installation.adoc [hunk 1]
```diff
@@ -13,7 +13,7 @@ If you are not connected to the internet or do not have access to online reposit
 
 Before installing {PlatformNameShort} on a disconnected network, you must meet the following prerequisites:
 
-. A created subscription manifest. See link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_operations_guide/assembly-aap-obtain-manifest-files#doc-wrapper[Obtaining a manifest file] for more information.
+. A subscription manifest that you can upload to the platform. See link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_operations_guide/assembly-aap-obtain-manifest-files#doc-wrapper[Obtaining a manifest file] for more information.
```
Comment (dcdacosta) [inline downstream/assemblies/platform/assembly-disconnected-installation.adoc:16]: FYI that this is going to be moved to the new Access management and authentication guide. Aine also set up attributes for the book URLs to make this easier going forward. So the link should be:
{URLCentralAuth}/index#assembly-aap-obtain-manifest-files

File: downstream/modules/platform/proc-completing-post-installation-tasks.adoc [hunk 2]
```diff
@@ -3,75 +3,17 @@
...
-----
-$ sudo -i
-# cat /etc/pulp/certs/root.crt
-(copy the contents of the file, including the lines with 'BEGIN CERTIFICATE' and
-'END CERTIFICATE')
-----
+Before your first login, you must add your subscription information to the platform. To obtain your subscription information in uploadable form, see link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html/red_hat_ansible_automation_platform_operations_guide/assembly-aap-obtain-manifest-files#assembly-aap-obtain-manifest-files[Obtaining a manifest file] in the Access management and authentication guide.
 
-. On the {ControllerName}:
+Once you have obtained your subscription manifest, see link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html/getting_started/with/Ansible_automation_platform[{TitleGettingStarted}] for instructions on how to upload your subscription information.
```
Comment (dcdacosta) [inline downstream/modules/platform/proc-completing-post-installation-tasks.adoc:11]: Update the link to use the new URL attribute for this title:
link:{URLGettingStarted}[{TitleGettingStarted}] for instructions...

File: downstream/modules/platform/proc-completing-post-installation-tasks.adoc [hunk 3]
```diff
@@ -3,75 +3,17 @@
...
-# vi /etc/pki/ca-trust/source/anchors/automationhub-root.crt
-(paste the contents of the root.crt file from the private automation hub into the new file and write to disk)
-# update-ca-trust
-----
+Now that you have successfully installed Ansible Automation Platform, to begin using its features, see the following guides for your next steps:
 
+link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html/getting_started/with/Ansible_automation_platform[{TitleGettingStarted}]
 
-.Additional Resources
+link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html/managing_content_in_automation_hub/index[Managing content in automation hub]
```
Comment (dcdacosta) [inline downstream/modules/platform/proc-completing-post-installation-tasks.adoc:17]: Use the newly created URL attribute:
link:{LinkHubManagingContent}

File: downstream/assemblies/platform/assembly-disconnected-installation.adoc (hunk 1 above)
Comment (rogrange) [reply downstream/assemblies/platform/assembly-disconnected-installation.adoc]: Change made.

**Review by dcdacosta — COMMENTED** [review]
Hey @rogrange, 

A few changes to use the new attributes that Aine created. Otherwise, LGTM!

File: downstream/modules/platform/proc-completing-post-installation-tasks.adoc (hunk 2 above)
Comment (rogrange) [reply downstream/modules/platform/proc-completing-post-installation-tasks.adoc]: Change made.

File: downstream/modules/platform/proc-completing-post-installation-tasks.adoc (hunk 3 above)
Comment (rogrange) [reply downstream/modules/platform/proc-completing-post-installation-tasks.adoc]: Change made.

File: downstream/modules/platform/proc-completing-post-installation-tasks.adoc [hunk 4]
```diff
@@ -3,75 +3,17 @@
...
-# vi /etc/pki/ca-trust/source/anchors/automationhub-root.crt
-(paste the contents of the root.crt file from the private automation hub into the new file and write to disk)
-# update-ca-trust
-----
+Now that you have successfully installed Ansible Automation Platform, to begin using its features, see the following guides for your next steps:
 
+link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html/getting_started/with/Ansible_automation_platform[{TitleGettingStarted}]
 
-.Additional Resources
+link:{LinkHubManagingContent}[Managing content in automation hub]
```
Comment (dcdacosta) [inline downstream/modules/platform/proc-completing-post-installation-tasks.adoc:17]: No need for the [Managing content in automation hub] it's included in the attribute.

Comment (rogrange) [reply downstream/modules/platform/proc-completing-post-installation-tasks.adoc]: Change made.

File: downstream/modules/platform/proc-completing-post-installation-tasks.adoc [hunk 5]
```diff
@@ -3,75 +3,17 @@
...
-----
-$ sudo -i
-# cat /etc/pulp/certs/root.crt
-(copy the contents of the file, including the lines with 'BEGIN CERTIFICATE' and
-'END CERTIFICATE')
-----
+Before your first login, you must add your subscription information to the platform. To obtain your subscription information in uploadable form, see link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html/red_hat_ansible_automation_platform_operations_guide/assembly-aap-obtain-manifest-files#assembly-aap-obtain-manifest-files[Obtaining a manifest file] in the Access management and authentication guide.
 
-. On the {ControllerName}:
+Once you have obtained your subscription manifest, see link:{URLGettingStarted}[{TitleGettingStarted}]for instructions on how to upload your subscription information.
```
Comment (dcdacosta) [inline downstream/modules/platform/proc-completing-post-installation-tasks.adoc:11]: You need a space between the close ] and "for". {URLGettingStarted}[{TitleGettingStarted}] for

File: downstream/modules/platform/proc-completing-post-installation-tasks.adoc [hunk 6]
```diff
@@ -3,75 +3,17 @@
...
 
-----
-$ sudo -i
-# vi /etc/pki/ca-trust/source/anchors/automationhub-root.crt
-(paste the contents of the root.crt file from the private automation hub into the new file and write to disk)
-# update-ca-trust
-----
+Now that you have successfully installed Ansible Automation Platform, to begin using its features, see the following guides for your next steps:
 
+link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html/getting_started/with/Ansible_automation_platform[{TitleGettingStarted}]
```
Comment (dcdacosta) [inline downstream/modules/platform/proc-completing-post-installation-tasks.adoc:15]: I missed this one the first time, sorry! 

{URLGettingStarted}[{TitleGettingStarted}]

File: downstream/modules/platform/proc-completing-post-installation-tasks.adoc [hunk 7]
```diff
@@ -3,75 +3,17 @@
...
 
-.Additional Resources
+link:{LinkHubManagingContent}
 
-* For further information on unknown certificate authority, see link:https://access.redhat.com/solutions/6707451[Project sync fails with unknown certificate authority error in {PlatformNameShort} 2.1].
+link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html/creating_and_consuming_execution_environments/index[Creating and consuming execution environments]
```
Comment (dcdacosta) [inline downstream/modules/platform/proc-completing-post-installation-tasks.adoc:19]: Gah, I missed another.... 
replace all of: link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html/creating_and_consuming_execution_environments/index[Creating and consuming execution environments]

with

link:{LinkBuilder}

File: downstream/modules/platform/proc-completing-post-installation-tasks.adoc (hunk 5 above)
Comment (rogrange) [reply downstream/modules/platform/proc-completing-post-installation-tasks.adoc]: Change made.

File: downstream/modules/platform/proc-completing-post-installation-tasks.adoc (hunk 6 above)
Comment (rogrange) [reply downstream/modules/platform/proc-completing-post-installation-tasks.adoc]: No worries. Change made.

File: downstream/modules/platform/proc-completing-post-installation-tasks.adoc (hunk 7 above)
Comment (rogrange) [reply downstream/modules/platform/proc-completing-post-installation-tasks.adoc]: Change made.

**Review by dcdacosta — APPROVED** [review]
LGTM!

**Pull request merged**